appserver/web/web-core/src/main/java/org/apache/catalina/connector/Request.java

```diff
@@ -55,7 +55,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-//Portions Copyright [2016-2022] [Payara Foundation]
+//Portions Copyright [2016-2024] [Payara Foundation]
```

Comment:
Slight amendment - missing "and/or its affiliates"
```suggestion
// Portions Copyright 2016-2024 Payara Foundation and/or its affiliates
```

Reply:
already done